site/README.md

```diff
@@ -0,0 +1,8 @@
+# Envoy site
+
+The following files are hosted on https://getenvoy.io, specifically via [Netlify redirects](https://github.com/tetratelabs/getenvoy.io/blob/master/site/static/_redirects):
+
+Latest master merge:
+* https://getenvoy.io/cli -> [./getenvoy_installer.sh](getenvoy_installer.sh)
```

Comment:
ps we can also call this "install.sh" similar to https://github.com/nvm-sh/nvm#install--update-script